saltstack/base/php/files/etc/php/7.2/mods-available/xdebug.ini

```diff
@@ -1,7 +1 @@
-zend_extension=xdebug.so
```

Comment:
This configuration is probably for older xdebug versions, if so keep the original, if not use the same as for the 8.0 one.

Reply:
ok reverting to :
xdebug.remote_enable=0
xdebug.remote_host=10.10.0.1
xdebug.remote_autostart=0
xdebug.remote_port=9000